src/jira.ts

```diff
@@ -174,11 +170,7 @@ Valid sample branch names:
   };
 
   /** Check if jira project validation is enabled then compare the issue project with the allowed statuses. */
-  static isProjectValid = (
-    shouldValidate: boolean,
-    allowedProjects: string[],
-    details: JIRADetails
-  ): boolean => {
+  static isProjectValid = (shouldValidate: boolean, allowedProjects: string[], details: JIRADetails): boolean => {
```

Comment:
Please check the project coding standards and flow it if possible

Reply:
Could you clarify what you think is wrong here? It is pretty much a copy-pasted definition from one of the previous functions, and then the npm prettify is sorting out the indentation et.c

Reply:
Yes, but with a different style.
you changed quoteProps style; I really didn't know what of them was matching the prettierrc config but you need to make sure you are confined prettier and eslint correctly
https://github.com/reliancehealthinc/action-jira-linter/blob/main/.prettierrc.js
https://github.com/reliancehealthinc/action-jira-linter/blob/main/.eslintrc.js